lib/orocos/log/task_context.rb

```diff
@@ -723,34 +720,37 @@ def port(name, verify = true)
                 end
             end
 
-            #Adds a new property or port to the TaskContext
+            # Register a property/port backed by a data stream
+            #
+            # @param [Pocolog::Datastream] stream the property/port type is autodetected
+            #   by the rock_stream_type metadata ('property' or 'port')
             #
             #* file_path = path of the log file
```

Comment:
file_path needs to be removed then from the documentation and an explanation for the second argument _backward is missing (same below)

Reply:
Since you are changing the interface here, where are the related tests and updates for the dependant functionalities?

Reply:
> Since you are changing the interface here, where are the related tests and updates for the dependant functionalities?

I don't see the relationship between the need for tests and the fact that it changes the interface (albeit trying to keep backward compatibility). Tests are needed, period.

Updated